### ISSUE TITLE
Support devtool modify --debug-build

### ISSUE THREAD

**Issue by afreof** [yoctoproject/vscode-bitbake#361]
Commit https://git.yoctoproject.org/poky/commit/?id=1f7eb26c947dde14698ae6e2ab88f0637027f4cb introduces a new command line option for devtool modify: --debug-build. It would be nice if the user interface also supported the --debug-build parameter. Maybe devtool modify should be callable with and without the --debug-build option.

**Comment by deribaucourt** [comment]
Hello @afreof !
Looks great. I don't want to bloat the interface too much. I'm considering just adding it by default whenever a "devtool modify" command is invoked. And maybe add a setting to disable this new behavior.

**Comment by afreof** [comment]
Just a note: Older devtool modify versions do not have the --debug-build option. Not sure how you can easily find out if the new feature is available or not.

**Comment by deribaucourt** [comment]
> Sounds good. Thank you.
> 
> Just a note: Older devtool modify versions do not have the --debug-build option. Not sure how you can easily find out if the new feature is available or not.

You're right! It's not a big deal though, we can run devtool --help to figure it out. We already do it to know if `devtool ide` is avilable. We should factorize both of these in a common call.
